.github/actions/run-interop-ping-test/action.yml

```diff
@@ -65,12 +65,12 @@ runs:
 
     - name: Build images
       working-directory: ${{ steps.find-workdir.outputs.WORK_DIR }}
-      run: make
+      run: make -j 2
```

Comment:
Is there a reason to still use make here? We could as well just use typescript for this step.

Going even further, we could make it a "proper" action that is published to the GitHub marketplace. That would allow us to move away from a `composite` one to either Docker (and allow us to write it in go or Rust) or JavaScript.

Reply:
To add another point, this is the root directory of these tests. Make here does nothing but call make in the implementation folders. I wouldn't want to remove this make. It's a good interface point such that implementations know they have to provide a Makefile that generates a image.json. The rest can be done however they want.